app/views.py

```diff
@@ -210,6 +234,11 @@ def handle_app(appname):
         return jsonify(**eval(line))
 
 
+@app.route('/<appname>/all')
```

Comment:
After chatting with @ryan-j-mci about this, I think I'm going to remove the /all functionality - don't think we need it.